docs/HowToUse.md

```diff
@@ -0,0 +1,28 @@
+# How to use?
+
+Let's compile an example by using ONE-vscode
+
+## Install toolchains
+
+First, you need ONE toolchains. Let's install ONE toolchains. (You should install the in-house backend to follow this.)
+
+![toolchain](https://user-images.githubusercontent.com/7223627/172834540-945ed5f9-82b0-4388-bd46-4ea10587d701.gif)
```

Comment:
/cc @jyoungyun I wonder you have better gif that shows entire flow of installation.

Reply:
Okay, I will make new gif for this page. :)

Reply:
Image is too big I think. :) We can change image size using `<img src="url" width=500 />` code.

Reply:
I set it 400. (500 is still big...)

Reply:
Hmm... in browser, 400 is too small...